src/main/java/com/chariotsolutions/meetupnotifier/MeetupNotifierImpl.java

```diff
@@ -6,41 +6,55 @@
 import com.chariotsolutions.meetupnotifier.meetup.api.GoogleConverter;
 import com.chariotsolutions.meetupnotifier.meetup.api.MeetupQuery;
 import com.chariotsolutions.meetupnotifier.meetup.api.MeetupQueryExecutor;
+import com.chariotsolutions.meetupnotifier.meetup.api.messages.Result;
 
-import com.chariotsolutions.meetupnotifier.meetup.api.messages.Results;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
 import org.springframework.scheduling.annotation.EnableScheduling;
 import org.springframework.stereotype.Component;
 
 import java.io.IOException;
 import java.util.List;
-import java.util.stream.Collectors;
+import javax.sql.DataSource;
 
 @Component
 @EnableScheduling
 public class MeetupNotifierImpl {
 
-  private static final Logger logger = LoggerFactory.getLogger(MeetupNotifierImpl.class);
+  @Autowired
+  DataSource ds;
 
   private String[] groups;
 
   public void queryAndNotify() throws IOException {
     for (String meetupGroupName : groups) {
-      Results results = new MeetupQueryExecutor().executeQuery(new MeetupQuery(meetupGroupName));
-
-      List<Event> events = results.getResults().stream()
-          .map(new GoogleConverter()::convertTo)
-          .collect(Collectors.toList());
-
-      for (Event googleEvent : events) {
-        App.cal.events().insert("primary", googleEvent).execute();
-        logger.debug("An event was added to the calendar successfully");
+      JdbcTemplate template = new JdbcTemplate(ds);
+      String metupInCalendarQuery = "SELECT * from id_table where meetupID=?";
```

Comment:
In fact I would create a separate persistence / DAO class that handles retrieving and saving this data.  "DAO" = Data Access Object.  Its responsible for storing/retrieving calendar events to the DB.  It encapsulates how the data is written to the DB and what DB you write to.  So in the future that could change underneath without affecting anything using this class.